### ISSUE TITLE
JSDoc is 2-years old, last generated Mar 15 2013

### ISSUE THREAD

**Issue by jasonsturges** [toji/gl-matrix#149]
JSDoc at [http://toji.github.io](http://toji.github.io/gl-matrix/docs/2.2.0/files.html) is reporting documentation was generated almost two years ago at Fri Mar 15 2013 22:21:14 GMT-0700 (PDT).

> ![image](https://cloud.githubusercontent.com/assets/1213591/6468133/9c162560-c198-11e4-928c-38e3baaec08b.png)

There are discrepancies, such as class [glMatrix](http://toji.github.io/gl-matrix/docs/2.2.0/symbols/glMatrix.html) file [common.js](http://toji.github.io/gl-matrix/docs/2.2.0/symbols/src/gl-matrix_src_gl-matrix_common.js.html) does not include `glMatrix.toRadian()` method.

**Comment by ToucheSir** [comment]
@toji Would you be open for a PR to update the online docs?

**Comment by toji** [comment]
I would be. I also need to figure out a more reasonable workflow for updating them so that we don't get so out of date.

**Comment by ToucheSir** [comment]
Ah, that's more than fair. Keeping documentation in sync with gh-pages does seem to be somewhat of a non-trivial matter. 

Thanks for the swift response! I hope to have some time to look at this soon.

**Comment by toji** [comment]
Updated the documentation this weekend, though it required me to stop using my old templates so the docs are a bit uglier now. That can be fixed down the road, though. Big change was that jsdoc could be installed and run as an npm package now instead of a Java applet like I was doing previously. Given that I've made re-generation of the docs part of the normal build process now, so we shouldn't get this out of date again.
